src/main/kotlin/no/nav/syfo/Bootstrap.kt

```diff
@@ -99,12 +99,12 @@ fun main() {
                 enforceCallId(NAV_CALLID)
```

Comment:
Jeg ser at dette her ikke er endret i denne pr'en, men jeg synes kanskje det hadde vært ryddigere om alt som går på oppsett av routing skjer i initRouting-metoden :)

Reply:
Ok. Det krever i såfall at initRouting tar databasen som en parameter, som gjør SelftestSpek litt klønete igjen. Men jeg kan løse det med å rydde naisRouting ut til en egen metode som kalles av initRouting.

Reply:
Ellers kan du rename initRouting til initRoutingNais da :)

Reply:
Synes det gir mening å ha en initRoutingmetode for all routing, men også ha separert naisRouting ut for å kunne teste det separat. Synes det ble ryddig nå :)